src/lib/es5.d.ts

```diff
@@ -2616,24 +2618,26 @@ interface Uint8ClampedArray {
     subarray(begin?: number, end?: number): Uint8ClampedArray;
 
     /**
-     * Converts a number to a string by using the current locale.
+     * Returns a string representation of the array in the current locale.
      */
     toLocaleString(): string;
 
     /**
-     * Returns a string representation of an array.
+     * Returns a string representation of the array.
      */
     toString(): string;
 
-    /** Returns the primitive value of the specified object. */
+    /**
+     * Returns the primitive value of the array.
+     */
     valueOf(): Uint8ClampedArray;
 
     [index: number]: number;
 }
 
 interface Uint8ClampedArrayConstructor {
     readonly prototype: Uint8ClampedArray;
-    new(length: number): Uint8ClampedArray;
+    new(length?: number): Uint8ClampedArray;

```

Comment:
I'm confused about these TypedArray types.

They are introduced in ES6 (ES2015), why are they in es5.d.ts? (Can we stop maintaining these old spec versions and merge them all)

This constructor without parameter is now declared in es2017.typedarrays.d.ts, but in es6 spec chapter [22.2.1.1 %TypedArray% ( )](https://262.ecma-international.org/6.0/#sec-%typedarray%-intrinsic-object-%typedarray%) they are already allowed.